internal/model/model.go

```diff
@@ -30,8 +35,29 @@ type Bookmark struct {
 
 // Account is person that allowed to access web interface.
 type Account struct {
-	ID       int    `db:"id"       json:"id"`
-	Username string `db:"username" json:"username"`
-	Password string `db:"password" json:"password,omitempty"`
-	Owner    bool   `db:"owner"    json:"owner"`
+	ID       int        `db:"id"                   json:"id"`
+	Username string     `db:"username"             json:"username"`
+	Password string     `db:"password"             json:"password,omitempty"`
+	Owner    bool       `db:"owner"                json:"owner"`
+	Config   UserConfig `db:"config"               json:"config"`
+}
+
+type UserConfig struct {
```

Comment:
If you implement [`driver.Valuer`](https://pkg.go.dev/database/sql/driver#Valuer) here, you can avoid (un)marshalling in the database logic:

```
func (c UserConfig) Value() (driver.Value, error) {
	return json.Marshal(c)
}
```

I'm sorry about this, I checked the example I provided and it was wrong. I made it really quick and forget that I left the marshalling logic instead of properly impementing the `Value() (driver.Value, error)` method.

With this, you can do:

```
db.Exec("UPDATE account SET config = ? WHERE username = ?", account.Config, account.Username)
```

directly, without manually marshalling anything.